clear-containers-image/debian.changelog

```diff
@@ -1,3 +1,19 @@
+clear-containers-image (18400-36) stable; urgency=medium
+
+  * Update clear-containers-image from 18220 to 18400.
+
+version: 18280
```

Comment:
All of this descriptive text needs to be indented as it is currently making this changelog file "invalid". See:

- https://www.debian.org/doc/debian-policy/#debian-changelog-debian-changelog

Note that you can check when the file is valid by doing:

```
$ sudo apt-get -y install dpkg-dev
$ dpkg-parsechangelog -l /path/to/your/changelog
```

If the `dpkg-parsechangelog` command outputs anything to `stderr`, the file is invalid (it seems that checking the exit code isn't reliable as the command attempts to recover and ignore bad input).